io/GltfWriter.cpp

```diff
@@ -75,55 +76,58 @@ void GltfWriter::addArgs(ProgramArgs& args)
     args.add("filename", "Output gltf filename", m_filename).setPositional();
     args.add("metallic", "Metallic factor [0-1]", m_metallic);
     args.add("roughness", "Roughness factor [0-1]", m_roughness);
-    args.add("red", "Red factor [0-1]", m_red);
-    args.add("green", "Green factor [0-1]", m_green);
-    args.add("blue", "Blue factor [0-1]", m_blue);
+    args.add("red", "Base red factor [0-1]", m_red);
```

Comment:
What does "Base" mean?

Reply:
I don't know. I read it somewhere. I have no idea how the renderer figures out what colors to use for which faces.